build.gradle

```diff
@@ -105,10 +105,13 @@ dependencies {
     testCompile(javadocJDKFiles)
 
     compile 'org.broadinstitute:barclay:1.0.0-24-g87c3fa2-SNAPSHOT'
-    compile ('com.intel:genomicsdb:0.3.0') {
+    compile ('com.intel:genomicsdb:' + genomicsdbVersion)  {
+        exclude module: 'spark-core_2.10'
         exclude module: 'htsjdk'
         exclude module: 'protobuf-java'
     }
+    compile 'com.googlecode.protobuf-java-format:protobuf-java-format:1.2'
+    compile 'com.googlecode.json-simple:json-simple:1.1.1'
```

Comment:
I don't see any uses of these.  Are they still necessary?  Are they transitive dependencies of genomicsDB? If so, why aren't they included automatically?